UI_UX/Frontend/src/styles/AppHeader.scss

```diff
@@ -1,21 +1,81 @@
-$headerHeight: 74px;
+$headerHeight: 90px;
 
 .AppHeader {
 	height: $headerHeight;
-	background-color: #200344;
+	background-image: linear-gradient(to right, #0b3f9c, #2e58db);;
     margin: 0px 0px;
     color: white;
 
     .AppHeader_logo {
         padding: 0 25px;
         margin: auto 0;
-        height: 47px;
+        height: 73px;
     }
 
-    .AppHeader_connect{
-    	position: relative;
+    .AppHeader_exchange_rate{
     	line-height: $headerHeight;
-    	padding-left: 10px;
-    	height: max-content;
+    	font-size: 14px;
+    }
+
+    .AppHeader_exchange_rate_loading{
+    	float: left;
+    	top: 27px;
+    	position: relative;
+    }
+
+    .AppHeader_account_info{
+    	position: absolute;
+    	right: 60px;
+    	height: $headerHeight;
+    }
+
+    .AppHeader_balance{
+    	text-align: right;
+    	height: $headerHeight;
+    	float: left;
+    	position: relative;
+    	margin-right: 20px;
+
+    	& p{
+    		font-size: 18px;
+    		padding-top: 20px;
+    		position: relative;
+    		top: 2px;
+    	}
+
+    	& span{
+    		font-size: 14px;
+    	}
+
+    	& .AppHeader_balance_loading{
+			top: 42px;
+		    position: absolute;
+		    right: 15px;
+    	}
+    }
+
+    .AppHeader_address{
+    	height: $headerHeight;
+    	float: left;
+
+    	& span{
+    		font-size: 14px;
+    		line-height: $headerHeight;
+    		padding-left: 10px;
+    	}
+
+    	& .paper{
+    		top: 14px;
+		 	position: relative;
```

Comment:
Make sure to use the same spacing for everywhere. Going to add an EditorConfig for this.

Reply:
fixed